plugins/jobs/src/js/components/form/helpers/JobFormData.ts

```diff
@@ -46,6 +46,12 @@ export interface FormOutput {
   disk: number;
   dockerParams: DockerParameter[];
   args: string[];
+  scheduleEnabled?: boolean;
```

Comment:
Is this really optional?

Reply:
Yes. A schedule can be submitted without the `enabled` property.